notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb

```diff
@@ -42,16 +1530,16 @@
         "id": "OjLCmM1RKCkf"
       },
       "source": [
-        "data = [[\"not sure\", \"maybe\"],\n",
-        "        [\"I may agree\", \"maybe\"],\n",
+        "data = [[\"not sure\", \"no\"],\n",
+        "        [\"I may agree\", \"no\"],\n",
         "        [\"I don't know\", \"no\"],\n",
         "        [\"I have no clue\", \"no\"],\n",
-        "        [\"I think so\", \"yes\"],\n",
-        "        [\"I agree\", \"yes\"],\n",
+        "        [\"I think so\", \"no\"],\n",
+        "        [\"I agree\", \"no\"],\n",
         "        [\"I'm not sure if I agree\", \"maybe\"],\n",
```

Comment:
Not sure if this was a maybe that was missed or if it was intentionally left in but I'd guess this should be a no instead of a maybe.  Please fix before final PR.